llama-index-core/llama_index/core/storage/chat_store/simple_chat_store.py

```diff
@@ -85,4 +85,4 @@ def from_persist_path(
             return cls()
         with fs.open(persist_path, "r") as f:
             data = json.load(f)
-        return cls.parse_raw(data)
+        return cls.parse_obj(data)
```

Comment:
Isn't this a breaking change? What about users who still have their data saved as a json string?

Reply:
It is breaking yes. However it is not very useful to not save a JSON object in a JSON file because then you can't use any JSON tool on it (like prettify) to inspect or edit the file.

Reply:
We should probably keep an option to try both parse_raw or parse_obj so as to not destroy peoples data

Reply:
yea the latter is what I meant 👍🏻 in the `from_persist_path()` method

Reply:
I added a check for it in the latest commit, what do you think of it?